ci/rpm7/build.sh

```diff
@@ -0,0 +1,370 @@
+if [ -n "${upload}" ]; then
+    if [ -z "${REPO_USER}" ] || [ -z "${REPO_PASSWORD}" ]; then
+        echo "Builder: failed, REPO_USER or REPO_PASSWORD env variables not set"; exit 1;
+    fi
+fi
+
+echo "Builder: check if target folder can be created"
+if [ -z "${path}" ]; then path='/opt/fiware-orion'; fi
+mkdir -p ${path} > /dev/null 2>&1
+if [ ! -d "${path}" ]; then echo "Builder: failed, not enough permissions"; exit 1; fi
+cd ${path}
+
+echo "Builder: if branch is empty, check if source exists"
+if [ -z "${branch}" ] && [ ! -f ${path}/LICENSE ]; then echo "Builder: failed, source code not found"; exit 1; fi
+
+
+echo "================================================================="
+echo "                             PREPARE                             "
+echo "================================================================="
+
+echo "Builder: create temp folders"
+rm -Rf /tmp/builder || true && mkdir -p /tmp/builder/{db1,db2,db,bu}
+
+if [ -n "${database}" ]; then
+    echo "Builder: starting Mongo"
+    mongod --dbpath /tmp/builder/db  --nojournal --quiet > /dev/null 2>&1 &
+    sleep 3
+    export MONGO_HOST=localhost
+fi
+
+if [ -n "${branch}" ]; then
+    echo "================================================================="
+    echo "                         CLONE                                   "
+    echo "================================================================="
+
+    echo "Builder: cloning branch: "${branch}
+    if [ -n "${source}" ]; then url_src=${source}; fi
+    echo "Builder: cloning from: "${url_src}
+    git clone -b ${branch} ${url_src} ${path}
+    chown -R root:root ${path}
+
+fi
+
+if [ -n "${make}" ]; then
+    echo "================================================================="
+    echo "                        BUILD                                    "
+    echo "================================================================="
+
+    _fix
+    echo "Builder: build ${make}"
+    make build_${make}
+    if [ $? -ne 0 ]; then echo "Builder: make failed"; exit 1; fi
+    _unfix
+
+fi
+
+if [ -n "${install}" ]; then
+    echo "================================================================="
+    echo "                        INSTALL                                  "
+    echo "================================================================="
+
+    _fix
+    make install
+    if [ $? -ne 0 ]; then echo "Builder: installation failed"; exit 1; fi
+    _unfix
+
+fi
+
+if [ -n "${execute}" ]; then
+    echo "================================================================="
+    echo "                        EXECUTE                                  "
+    echo "================================================================="
+
+    _execute
+    exit 0
+
+fi
+
+if [ -n "${optional}" ]; then
+    echo "================================================================="
+    echo "                       OPTIONAL TESTS                            "
```

Comment:
OPTIONAL -> COMPLIANCE

Reply:
fixed in https://github.com/telefonicaid/fiware-orion/pull/3192/commits/85f9adb0d02f1a9d1906bea2198af60546f9b612